From c9b74bceb38b662c1e112611b45e389e9d38b22b Mon Sep 17 00:00:00 2001
From: Fabian Vogt <fvogt@suse.de>
Date: Mon, 3 Apr 2023 12:45:02 +0200
Subject: [PATCH] Revert "QGnomeTheme: Allow Space, Return, Enter and Select to
 press buttons"

Workaround for https://bugreports.qt.io/browse/QTBUG-112523

This reverts commit f2eab01eb9f95f9eebc10bbe96e6c4810cd654b1.
---
 src/platformsupport/themes/genericunix/qgenericunixthemes.cpp | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/platformsupport/themes/genericunix/qgenericunixthemes.cpp b/src/platformsupport/themes/genericunix/qgenericunixthemes.cpp
index 6e01af052c..cb1b39db64 100644
--- a/src/platformsupport/themes/genericunix/qgenericunixthemes.cpp
+++ b/src/platformsupport/themes/genericunix/qgenericunixthemes.cpp
@@ -755,9 +755,6 @@ QVariant QGnomeTheme::themeHint(QPlatformTheme::ThemeHint hint) const
         return QVariant(QChar(0x2022));
     case QPlatformTheme::UiEffects:
         return QVariant(int(HoverEffect));
-    case QPlatformTheme::ButtonPressKeys:
-        return QVariant::fromValue(
-                QList<Qt::Key>({ Qt::Key_Space, Qt::Key_Return, Qt::Key_Enter, Qt::Key_Select }));
     default:
         break;
     }
-- 
2.40.0

